.github/workflows/ci.yml

```diff
@@ -25,13 +25,4 @@ jobs:
           node-version: '10'
       - run: npm ci
       - run: npm run front-end:lint
-      - run: npm run back-end:lint
-  audit:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
-        with:
-          node-version: '10'
-      - run: npm ci
-      - run: npm audit --audit-level=moderate
\ No newline at end of file
+      - run: npm run back-end:lint
```

Comment:
for future consideration: [end-of-file-fixer](https://github.com/pre-commit/pre-commit-hooks#end-of-file-fixer)
> Makes sure files end in a newline and only a newline.

Reply:
Split to #7